include/sequence.h

```diff
@@ -3,132 +3,131 @@
+#define NA_BGM_MOUNTAIN_REGION 0x13         // Mountain Village ("SNOW_AREA")
+#define NA_BGM_PIRATES_FORTRESS 0x14        // Pirates' Fortress ("PIRATE_AREA")
+#define NA_BGM_CLOCK_TOWN_DAY_1 0x15        // Clock Town Day 1. This song's filename is dummied out in MM3D ("DUMMY_21")
+#define NA_BGM_CLOCK_TOWN_DAY_2 0x16        // Clock Town Day 2. This song's filename is dummied out in MM3D ("DUMMY_22")
+#define NA_BGM_CLOCK_TOWN_DAY_3 0x17        // Clock Town Day 3. This song's filename is dummied out in MM3D ("DUMMY_23")
+#define NA_BGM_FILE_SELECT 0x18             // File Select ("FILE_SELECT")
+#define NA_BGM_CLEAR_EVENT 0x19             // Event Clear ("EVENT_CLEAR")
+#define NA_BGM_ENEMY 0x1A                   // Battle ("ENEMY")
+#define NA_BGM_BOSS 0x1B                    // Boss Battle ("BOSS00")
+#define NA_BGM_WOODFALL_TEMPLE 0x1C         // Woodfall Temple ("JUNGLE_DUNGEON")
+#define NA_BGM_MARKET 0x1D                  // Clock Town Day 1. Duplicate of #15. Name comes from MM3D. ("MARKET")
+#define NA_BGM_OPENING 0x1E                 // Forest Ambush ("OPENING")
+#define NA_BGM_INSIDE_A_HOUSE 0x1F          // House ("LINK_HOUSE")
+#define NA_BGM_GAME_OVER 0x20               // Game Over ("GAME_OVER")
+#define NA_BGM_CLEAR_BOSS 0x21              // Boss Clear ("BOSS_CLEAR")
+#define NA_BGM_GET_ITEM 0x22                // Item Obtained ("ITEM_GET")
+#define NA_BGM_GATE_OPEN 0x23               // Clock Town Day 2. Duplicate of #16. In MM3D, it shares its filename with the Door of Time song from OoT ("GATE_OPEN")
+#define NA_BGM_GET_HEART 0x24               // Complete a Heart Piece ("HEART_GET")
+#define NA_BGM_TIMED_MINI_GAME 0x25         // Playing a timed Minigame ("MINI_GAME_2")
+#define NA_BGM_GORON_RACE 0x26              // Goron Race ("GORON_RACE")
+#define NA_BGM_MUSIC_BOX_HOUSE 0x27         // Music Box House ("ORGEL_HOUSE")
+#define NA_BGM_FAIRY_FOUNTAIN 0x28          // Fairy's Fountain. ("GODESS")
+#define NA_BGM_ZELDAS_LULLABY 0x29          // Zelda's Lullaby ("HIME")
+#define NA_BGM_ROSA_SISTERS 0x2A            // Rosa Sisters' Dance ("SISTER_DANCER")
+#define NA_BGM_OPEN_CHEST 0x2B              // Open Chest ("OPEN_TRE_BOX")
+#define NA_BGM_MARINE_RESEARCH_LAB 0x2C     // Marine Research Laboratory ("DRUGSTORE")
+#define NA_BGM_GIANTS_THEME 0x2D            // The Four Giants ("KYOJIN_ROOM")
+#define NA_BGM_SONG_OF_STORMS 0x2E          // Guru-Guru's Song ("FUSHA")
+#define NA_BGM_ROMANI_RANCH 0x2F            // Romani Ranch ("RONRON")
+#define NA_BGM_GORON_VILLAGE 0x30           // Goron Village ("GORON")
+#define NA_BGM_MAYORS_OFFICE 0x31           // Mayor Dotour ("MEETING")
+#define NA_BGM_OCA_EPONA 0x32               // Ocarina Epona's Song ("OCA_EPONA")
+#define NA_BGM_OCA_SUNS 0x33                // Ocarina Sun's Song ("OCA_SUNS")
+#define NA_BGM_OCA_TIME 0x34                // Ocarina Song of Time ("TIME_NORMAL")
+#define NA_BGM_OCA_STORM 0x35               // Ocarina Song of Storms ("OCA_STORM")
```

Comment:
I kinda want OCA to be full length OCARINA, we aren't saving that much but for some reason this shorthand takes me a good 5 seconds to remember OCA is short for ocarina, is that just me tho?

Reply:
It's not just you.

Reply:
I have no attachments to OCA. I only did it to keep the change minimal from OG. Feel free to change if you like